tests/unit/test_irc.py

```diff
@@ -126,22 +116,23 @@ def test_prepare_does_not_copy_files_if_already_exist(irc_bridge_service, mocker
     systemd.daemon_reload, and systemd.service_enable methods were called exactly once.
     """
     mock_install_snap_package = mocker.patch.object(irc_bridge_service, "_install_snap_package")
+    mock_generate_media_proxy_key = mocker.patch.object(
+        irc_bridge_service, "_generate_media_proxy_key"
+    )
     mock_copy = mocker.patch.object(shutil, "copy")
     mock_mkdir = mocker.patch.object(pathlib.Path, "mkdir")
-    mock_daemon_reload = mocker.patch.object(systemd, "daemon_reload")
-    mock_service_enable = mocker.patch.object(systemd, "service_enable")
 
     mocker.patch.object(pathlib.Path, "exists", return_value=True)
 
-    irc_bridge_service.prepare()
+    with patch("builtins.open", mock_open(read_data="config")):
```

Comment:
It is difficult to see what we are mocking here. Not sure how to do it, maybe some way using tmp_path someway and not mock all filesystem calls, like copy (that btw uses open internally several times), mkdir, open...?

Reply:
I'll close this one so we can discuss in the next ok? I believe is the same problem.